lineapy/api/api.py

```diff
@@ -292,6 +289,39 @@ def get(artifact_name: str, version: Optional[int] = None) -> LineaArtifact:
     return linea_artifact
 
 
+def get_pipeline(name: str) -> Pipeline:
+
+    execution_context = get_context()
+    db = execution_context.executor.db
+
+    pipeline_orm = db.get_pipeline_by_name(name)
+
+    artifact_names = [
+        artifact.name
+        for artifact in pipeline_orm.artifacts
+        if artifact.name is not None
+    ]
+
+    dependencies = dict()
+    for dep_orm in pipeline_orm.dependencies:
+        post_artifact = dep_orm.post_artifact
+        if post_artifact is None:
+            continue
```

Comment:
when would this happen? i would expect that if an entry is in the dependencies, it will not be none.